test.sh

```diff
@@ -0,0 +1,41 @@
+PLATFORM=$1
+TYPE=$2
+WHERE="cat != ManualTest"
+TEST_DIR="."
+TEST_PATTERN="*Test.dll"
+ASSEMBLIES=""
+
+if [ -d "$TEST_DIR/_tests" ]; then
+  TEST_DIR="$TEST_DIR/_tests"
+fi
+
+NUNIT="$TEST_DIR/NUnit.ConsoleRunner.3.2.0/tools/nunit3-console.exe"
+NUNIT_COMMAND="$NUNIT"
+NUNIT_PARAMS="--teamcity"
+
+if [ "$PLATFORM" = "Windows" ]; then
+  WHERE="$WHERE && cat != LINUX"
+elif [ "$PLATFORM" = "Linux" ]; then
```

Comment:
Yeah... I wasn't sure, I dislike "mono" as a platform when Windows is the alternative. I'd be in favour of having "Mac" or "OSX" added as well in case we need to handle something differently there/add more Mac specific tests.